t/page/countries.rb

```diff
@@ -2,25 +2,27 @@
 require_relative '../../lib/page/countries'
 require 'pry'
 
-CJSON = 'https://cdn.rawgit.com/everypolitician/everypolitician-data/%s/countries.json'.freeze
-SHA   = 'd8a4682f'.freeze
+CJSON      = 'https://cdn.rawgit.com/everypolitician/everypolitician-data/%s/countries.json'.freeze
+SHA        = 'd8a4682f'.freeze
+datasource = CJSON % SHA
 
 describe 'Countries' do
   subject do
     Page::Countries.new
   end
 
   it 'has a list of all countries in Everypolitician' do
-    assert_equal subject.countries.first[:name], 'Abkhazia'
+    Everypolitician.countries_json = datasource
```

Comment:
Ah, when I said before to make all tests use the same setup, I meant in a `before`:

```ruby
  before { Everypolitician.countries_json = CJSON % SHA }
```